classes/class-wc-connect-help-provider.php

```diff
@@ -249,7 +249,113 @@ protected function get_health_items() {
 		}
 
 		protected function get_services_items() {
-			return array();
+			$service_items = array();
+
+			$enabled_services = $this->service_settings_store->get_enabled_services();
+
+			if ( empty( $enabled_services ) ) {
+				$service_items[] =	(object) array(
+					'key' => 'wcc_services_empty',
+					'type' => 'text',
+					'class' => 'form_text_body_copy',
+					'value' => __( 'No services have been enabled', 'woocommerce' )
+				);
+
+				return $service_items;
+			}
+
+			foreach ( (array) $enabled_services as $enabled_service ) {
+				$indicator_key = "{$enabled_service->method_id}_{$enabled_service->instance_id}_1";
+				$last_failed_request_timestamp = intval( get_option( 'wc_connect_last_rate_request_failure', -1 ) );
+
+				$service_settings_url = esc_url( add_query_arg(
+					array(
+						'page' => 'wc-settings',
+						'tab' => 'shipping',
+						'instance_id' => $enabled_service->instance_id
+					),
+					admin_url( 'admin.php' )
+				) );
+
+				$review_link = sprintf(
+					wp_kses(
+						__( '<a href="%s">Review service settings</a>', 'woocommerce' ),
+						array(  'a' => array( 'href' => array() ) )
+					),
+					esc_url( $service_settings_url )
+				);
+
+				$edit_link = sprintf(
+					wp_kses(
+						__( '<a href="%s">Edit service settings</a>', 'woocommerce' ),
+						array(  'a' => array( 'href' => array() ) )
+					),
+					esc_url( $service_settings_url )
+				);
+
+				$ago_edit_link = sprintf(
+					wp_kses(
+						_x( 'Request was made %1$s ago - <a href="%2$s">edit service settings</a>', 'e.g. two hours', 'woocommerce' ),
+						array( 'a' => array( 'href' => array() ) )
+					),
+					esc_html( human_time_diff( current_time( 'timestamp' ) - $last_failed_request_timestamp ) ),
+					esc_url( $service_settings_url )
+				);
+
+				// Figure out if the service has any settings saved at all
+				$service_settings = $this->service_settings_store->get_service_settings( $enabled_service->method_id, $enabled_service->instance_id );
+				if ( empty( $service_settings ) ) {
+					$indicator = $this->build_indicator(
+						$indicator_key,
+						'notice',
+						'indicator-error',
+						__( 'Setup for this service has not yet been completed', 'woocommerce' ),
```

Comment:
I don't think we should. They may not want to use shipping rate services at all.  They may only want shipping labels out of WCC for example and offer free or flat rate shipping in all zones.